lib/utils/checkInvalidCLIOptions.js

```diff
@@ -45,6 +44,22 @@ const suggest = (all, invalid) => {
 	return null;
 };
 
+/**
+ * Converts a string to kebab case.
+ * For example, `kebabCase('oneTwoThree') === 'one-two-three'`.
+ * @param {string} opt
+ * @returns {string}
+ */
+const kebabCase = (opt) => {
+	const matches = opt.match(/[A-Z]?[a-z]+|[A-Z]|[0-9]+/g);
+
+	if (matches) {
+		return matches.map((s) => s.toLowerCase()).join('-');
+	}
+
+	return '';
+};
```

Comment:
[suggestion] Can we use a simpler version according to the following `kebab-case` package?

https://github.com/joakimbeng/kebab-case/blob/21cba70cd5e694470cc01402b9df829bec1cada7/index.js#L5-L9

Reply:
The linked code is only a little bit simpler, and I think my code is better at handling edge cases.

My function:
```
> kebabCase('abcDef')
'abc-def'
> kebabCase('AbcDef')
'abc-def'
> kebabCase('Option2')
'option-2'
> kebabCase('Abc123defG')
'abc-123-def-g'
```

The linked function:
```
> kebabCase('abcDef')
'abc-def'
> kebabCase('AbcDef')
'-abc-def'
> kebabCase('Option2')
'-option2'
> kebabCase('Abc123defG')
'-abc123def-g'
```

Reply:
I see 👍🏼